package.json

```diff
@@ -63,7 +63,7 @@
     "@blackbaud/skyux-logger": "1.1.1",
     "@ngtools/webpack": "7.3.9",
     "@pact-foundation/karma-pact": "2.2.0",
-    "@pact-foundation/pact-node": "8.6.0",
+    "@pact-foundation/pact-node": "9.0.0",
```

Comment:
You think this breaking change will affect us? https://github.com/pact-foundation/pact-node/blob/master/CHANGELOG.md#features

Reply:
Great question.  I'm not anticipating any breaking changes given our usage of that library.